docs/training_guides/etl_best_practices.rst

```diff
@@ -0,0 +1,558 @@
+==================================
+Introduction to ETL Best Practices
+==================================
+
+This training guide will walk you through some basic ETL workflows using Parsons. It is based off of a two-part training designed by Cormac Martinez del Rio and Shauna Gordon-McKeon.
+
+It introduces the basic concepts behind the Extract, Transform and Load (ETL) process by working through two examples. First, we focus on how to write a basic Parsons script that moves data between one platform (Mobilize) and another (Google Sheets). Then, we introduce some more advanced concepts such as data warehouses, platforms like Civis, and the use of log tables and schedulers to make your workflow easier to run and debug.
+
+.. contents:: Table of Contents
+    :depth: 3
+
+You can suggest improvements to this guide or request additional guides by filing an issue in our issue tracker or telling us in Slack. To get added to our Slack, and/or to request access to the recordings of this training, email us at *engineering@movementcooperative.org*.
+
+********
+Part One
+********
+
+^^^^^^^^^^^^
+Introduction
+^^^^^^^^^^^^
+
+Have you ever tried to get data from one platform or system to another? Doing so can be quite a pain, especially if you have to manually download and upload individual files. Luckily, websites and apps often have APIs, which stands for application programming interfaces. These are gateways that let us move data into and out of a system using code.
+
+**ETL**, which stands for Extract, Transform, and Load is the process by which we extract data from one systems API, transform that data so that it’s compatible with a second system's API, and then loading that data into that second system. 
+
+The format that source system gives us data is often different from the format that another system likes receiving data! Everyone thinks their way is best.
+Not so good for the movement, good for data engineers' job security. ;)
+
+Parsons can help us with every step of the way. We can use parsons to extract, transform, and load data! 
+
+Today we're going to be using a Parsons script to move data from Mobilize to Github. Our inspiration was an experience Shauna had managing canvassing volunteers who had signed up on Mobilize, but whose canvassing status was being tracked via Google Sheets. 
```

Comment:
I think you meant Mobilize to Google Sheets ;)

Reply:
Good catch!